lua/wikis/commons/Infobox/Person.lua

```diff
@@ -279,7 +352,11 @@ function Person:_setLpdbData(args, links, status, personType)
 		teampagename = mw.ext.TeamLiquidIntegration.resolve_redirect(teamLink or team or ''):gsub(' ', '_'),
 		teamtemplate = teamTemplate,
 		status = status,
-		type = personType,
+		type = self:_isPlayerOrStaff(),
+		role = self.role or {},
+		role2 = self.role2 or {},
+		role3 = self.role3 or {},
+		roles = self.roles or {},
```

Comment:
```suggestion
		type = personType,
```
should still be in extradata, i.e. a few lines further down (github doesn't let me comment on the lines though ...)